discovery-provider/src/queries/get_repost_feed_for_user.py

```diff
@@ -12,6 +12,10 @@ def get_repost_feed_for_user(user_id, args):
     feed_results = {}
     db = get_db_read_replica()
     with db.scoped_session() as session:
+        if "handle" in args:
+            handle = args.get("handle")
+            user_id = session.query(User.user_id).filter(User.handle == handle).first()
```

Comment:
I would maybe use the `handle_lc` field on the user and call .lower() on the input handle.

Reply:
GOOD EYE. Fixed here & other query. ty